index.js

```diff
@@ -0,0 +1,207 @@
+const select = document.querySelector("select");
+const section = document.querySelector("#display-info");
+const form = document.querySelector("form");
+
+const getFilms = async () => {
+  try {
+    const res = await axios.get("https://ghibliapi.herokuapp.com/films");
+    debugger;
+    res.data.forEach((film) => {
+      const option = document.createElement("option");
+      option.innerText = film.title;
+      select.appendChild(option);
+      debugger;
+    });
+  } catch (error) {
+    console.log(error);
+  }
+};
+getFilms();
+
+// const createSection = async () => {
+//   try {
+//     const res = await axios.get("https://ghibliapi.herokuapp.com/films");
+//     res.data.forEach((film) => {
+//   debugger
+//       const title = document.createElement("h3");
+//       const releaseYear = document.createElement("p");
+//       const description = document.createElement("p");
+//       title.innerText = film.title;
+//       releaseYear.innerText = film.release_date;
+//       description.innerText = film.description;
+//       section.appendChild(title);
+//       section.appendChild(releaseYear);
+//       section.appendChild(description);
+//     //   section.innerText = "";
+//     });
+//   } catch (error) {
+//     console.log(error);
+//   }
+// };
+
+// createSection();
+
+select.addEventListener("click", async (e) => {
+    debugger
+    try {
+        const res = await axios.get("https://ghibliapi.herokuapp.com/films");
+        res.data.forEach((film) => {
+      debugger
+          const title = document.createElement("h3");
+          const releaseYear = document.createElement("p");
+          const description = document.createElement("p");
+          title.innerText = film.title;
+          releaseYear.innerText = film.release_date;
+          description.innerText = film.description;
+          section.appendChild(title);
+          section.appendChild(releaseYear);
+          section.appendChild(description);
+        //   section.replace(title);
+        //   section.innerText = "";
+        });
+      } catch (error) {
+        console.log(error);
+      }
+});
+
+form.addEventListener("click", async (e) => {
+    e.preventDefault();
+    debugger
+    try {
+        const res = await axios.get("https://ghibliapi.herokuapp.com/films");
+        res.data.forEach((film) => {
+        const textInput = document.querySelector("#review-text-input");
+        debugger
+        const reviewsUl = document.querySelector("#reviews")
+        const reviewLi = document.createElement("li");
+        reviewLi.innerHTML = `<b>${film.title}</b> ${textInput.value}`
+        debugger
+        reviewsUl.appendChild(reviewLi);
+        textInput.value = "";
+    });
+    } catch (error) {
+        console.log(error);
+    }
+   
+});
```

Comment:
The event listener for the form should be `submit`.
In the form submission there's no need to call the api anymore. All we need to do is get the value in the select variable => `select.value` and the text from the input `input.value` and append that to the `ul`